blazor/combobox/form-validation.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: Form Validation in Blazor ComboBox Component | Syncfusion
+description: Checkout and learn about Form validation with Blazor ComboBox component in Blazor Sever App and Blazor WebAssembly App.
+platform: Blazor
+control: ComboBox
+documentation: ug
+---
+
+# Form Validation in ComboBox
+
+## ComboBox inside edit form
+
+The `EditForm` validates all data annotation rules using the `DataAnnotationsValidator`. If the input is valid, the form will be ready to be submitted. If the input is invalid, an error message will be displayed until a valid value is chosen.

```

Comment:
Content is generic and explain about input in editform, modify it to be more specific